### PR TITLE
[Fixes #112] Make many things scale better on mobile

### LINKED ISSUE #112: Display issues on mobile (iPhone 5)
dttw-iphone5

### PR DESCRIPTION
**Pull request by bluepichu**
Title.

Confirmed fix to #112 by reporter.

### CODE REVIEW

**Comment by zwade** [comment]
Rip my whitespace

XD sorry

**Comment by bluepichu** [comment]
Heheh, I have my sublime set to KILL TRAILING WHITESPACE WITH EXTREME PREJUDICE.

**Comment by zwade** [comment]
Cool, I might be able to look once I get home, but also packing

**Comment by bluepichu** [comment]
Oh... I guess reassign if you're taking this over.

Also, FYI, I had to push this live to let my friend to test that the fix actually worked, so there's no rush here.  It's already up.

**Comment by zwade** [comment]
Oh lol didn't realize I wasn't assigned. Nah Ellis can keep it

File: client/scss/media.scss [hunk 1]
```diff
@@ -10,29 +10,40 @@
 		&>a {
 			padding: 0px;
 			justify-content: center;
-			
+
 			h1 {
 				width: auto;
+				font-size: 12vw;
```
Comment (el1t) [inline client/scss/media.scss:16]: 😦

File: client/scss/media.scss [hunk 2]
```diff
@@ -10,29 +10,40 @@
...
+
 		nav {
 			padding: 0px;
 			justify-content: center;
 			border-bottom-color: $color-text-main;
 			width: 100%;
 			margin-left: 0px;
+
+			a {
+				width: 100px;
```
Comment (el1t) [inline client/scss/media.scss]: Any reason for this number? Could it be % or vw or something?

Comment (bluepichu) [inline client/scss/media.scss]: It's arbitrary to add more spacing between the header links ("home", "about", "contact") on mobile.  `em` might be a better candidate, but definitely not `%` or `vw`.

File: client/scss/media.scss (hunk 1 above)
Comment (bluepichu) [inline client/scss/media.scss:16]: I don't understand this comment.  Scale down the font size of the stylized site title to make it fit the width.

Comment (el1t) [inline client/scss/media.scss:16]: Eh I guess that works

**Comment by el1t** [comment]
lgtm 🎉

**Comment by el1t** [comment]
Hey I was gonna do that :cry:

**Pull request merged**